.github/workflows/ci.yaml

```diff
@@ -28,8 +28,6 @@ jobs:
             ROS_DISTRO: humble
           - IMAGE: jazzy-ci
             ROS_DISTRO: jazzy
-          - IMAGE: jazzy-ci-testing
-            ROS_DISTRO: jazzy
```

Comment:
Wait - should we leave this part? So that we are still testing main branch against Jazzy?

Reply:
I was wondering about this -- what is the difference between `jazzy-ci` and `jazzy-ci-testing`? Because I just removed the latter

Reply:
Ah it's for the Docker images

Regular builds a docker `FROM ros:${ROS_DISTRO}-ros-base`

Testing builds it `FROM osrf/ros2:testing`

... so it seems like the testing repo should only really be happening on rolling if I understand correctly?

Reply:
Ah yes, I saw the humble-ci above, but missed that there was still a jazzy-ci above. All good.

Reply:
It does raise the question of adding a `rolling-ci-testing` job though?

Reply:
I'm guessing the thought is that rolling-testing is just too volatile and things would be breaking all the time rather than every sync?

Reply:
Yeah, I guess we can always add jobs back as needed. Thanks for checking!